azure-pipelines.yml

```
@@ -298,17 +298,17 @@ stages:
             continueOnError: true
             condition: always()
 
-        #- job: Linux_FCS
-        #  pool:
-        #    vmImage: ubuntu-16.04
-        #  variables:
-        #  - name: _SignType
-        #    value: Test
-        #  steps:
-        #  - checkout: self
-        #    clean: true
-        #  - script: ./fcs/build.sh Build
-        #    displayName: Build
+        - job: Linux_FCS
+          pool:
+            vmImage: ubuntu-16.04
+          variables:
+          - name: _SignType
+            value: Test
+          steps:
+          - checkout: self
+            clean: true
+          - script: ./fcs/build.sh Build
```

```suggestion
          - script: ./fcs/build.sh
```

And the same for the others I suppose

---

Done, good idea. The Windows job runs tests and tests packaging as well, which we may want to consider doing on all of them now that it's more unified.